src/System.Composition.AttributedModel/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Resources;
```

Comment:
Are such AssemblyInfo.cs files needed?  I was under the impression that the buildtools supplied such attributes automatically.